css/style.css

```
@@ -0,0 +1,154 @@
+/*=============================================================================
+ベース
+==============================================================================*/
+*, *::before, *::after {
+  box-sizing:inherit;
+}
+
+html {
+  box-sizing:border-box;
+}
+
+body {
+  color:#333;
+  font-family:'Roboto','sans-serif';
+  line-height:1.8;
+  font-size:14px;
+}
+
+img {
+  width:100%;
+  height:auto;
+  vertical-align:bottom;
+}
+
+a {
+  text-decoration:none;
+  color:inherit;
+}
+
+a:hover {
+  opacity:0.7;
+}
+/*=============================================================================
+グローバル
+==============================================================================*/
+.btn {
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  background-color:#ffa800;
+  color:#fff;
+  border-radius:9999px;
+  cursor:pointer;
+  font-weight:bold;
+  margin:0 auto;
+}
+
+.pcNone {
+  display:none;
+}
+
+/*============================================================================*/
+header {
+  background-color:#fff;
+  position:sticky;
+  top:0;
+  box-shadow:rgba(0, 0, 0, 0.3) 1px 1px 3px;
+}
+
+header .inner {
+  max-width:980px;
+  width:96%;
+  height:60px;
+  display:flex;
+  justify-content:space-between;
+  align-items:center;
+  margin:auto;
+}
+
+header .logo a img {
+  display:flex;
+  width:100px;
+}
+
+header .nav .list {
+  display:flex;
+  align-items:center;
+}
+
+header .nav .list .item + .item {
+  margin-left:20px;
+}
+
+header .btn {
+  width:132px;
+  height:30px;
+}
+
+header .inner p  {
```

HTMLの構造に依存しすぎなので、pタグの部分にクラス指定して
```suggestion
header .toggle  {
```
とするといい気がします。

---

> HTMLの構造に依存しすぎなので、pタグの部分にクラス指定して
> 
> とするといい気がします。

修正しました。
e4568f8128205e30d1adcf228eb98f4eba5a08bc